beetsplug/replaygain.py

```diff
@@ -905,21 +928,55 @@ def handle_track(self, item, write):
 
         self._log.info(u'analyzing {0}', item)
 
-        try:
-            track_gains = self.backend_instance.compute_track_gain([item])
-            if len(track_gains) != 1:
-                raise ReplayGainError(
-                    u"ReplayGain backend failed for track {0}".format(item)
+        self._log.debug(u'format: {0}', item.format)
+
+        if item.format == 'Opus':
```

Comment:
Maybe some manner of global function to make this decision (e.g., `should_use_r128`) would be useful to encapsulate the policy here?